core/src/main/resources/META-INF/services/io.grpc.NameResolverProvider

```diff
@@ -0,0 +1 @@
+io.grpc.internal.DnsNameResolverProvider
```

Comment:
There can be multiple instances of this file. All the lines from all the files are taken as a whole, and the order the files are loaded in aren't well-defined (generally deterministic, but maybe the matter the order of the jars in the classpath matters and such).